README.md

```
@@ -40,105 +40,145 @@ add_filter( 'fen_ignore_error', 'fen_ignore_error', 10, 2 );
-## Installation ##
+## Installation
 
 1. You can clone the GitHub repository: `https://github.com/verygoodplugins/fatal-error-notify.git`
 2. Or download it directly as a ZIP file: `https://github.com/verygoodplugins/fatal-error-notify/archive/master.zip`
 
 This will download the latest copy of Fatal Error Notify.
 
-## Bugs ##
+## Bugs
+
 If you find an issue, let us know [here](https://github.com/verygoodplugins/fatal-error-notify/issues?state=open)!
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)